src/gui/plugins/spawn/Spawn.cc

```diff
@@ -190,7 +190,7 @@ void Spawn::LoadConfig(const tinyxml2::XMLElement *)
 }
 
 
-// TODO(ahcorde): Replace this when this function is on ign-rendering6
+// TODO(ahcorde): Replace this when this function is on ign-rendering7
```

Comment:
Another change that I think can be reverted. The function is already on `ign-rendering6`.

Reply:
reverted. c0657e52630f5020644d814ca7f2a391d44d9f49